gradle.properties

```diff
@@ -1,18 +1,19 @@
-# Project-wide Gradle settings.
```

Comment:
Normally you don't remove this file because it can contain app version (in case of library) or connection info for jenkins build. I don't see the harm on having this file in here because it doesn't change